regulations/urls.py

```diff
@@ -45,6 +45,7 @@
     # about page
     url(r'^about$', about, name='about'),
     url(r'^comments/attachment$', comment.upload_proxy),
+    url(r'^comments/attachment/preview$', comment.download_proxy),
```

Comment:
What do you think about making this a RESTful url, instead? Something like `comments/attachment/<key>`